cmd/eval.go

```diff
@@ -279,6 +296,9 @@ access.
 	evalCommand.Flags().BoolVarP(&params.failDefined, "fail-defined", "", false, "exits with non-zero exit code on defined/non-empty result and errors")
 	evalCommand.Flags().DurationVar(&params.timeout, "timeout", 0, "set eval timeout (default unlimited)")
 
+	evalCommand.Flags().IntVarP(&params.optimizationLevel, "optimize", "O", 0, "set optimization level")
+	evalCommand.Flags().VarP(&params.entrypoints, "entrypoint", "e", "set slash separated entrypoint path")
```

Comment:
Would it be feasible to derive that from the evaluated query? It's mandatory to provide a query anyways, so, I guess we could try to convert that to an entrypoint; or add a stub module:

```
opa eval -O2 -d foo.rego 'data.foo.baz = 1'
```
⏬ 
```rego
package __entrypoint__
e {
  data.foo.baz = 1
}
```
so we'd use `entrypoint/e` as entrypoint? Or maybe that's what some default logic could be, and `-e` would let you override it? 🤔